### ISSUE TITLE
EngineCrash after reloading rmlui widget

### ISSUE THREAD

**Issue by Beherith** [beyond-all-reason/spring#1832]
<html><body>

module | address | file | line
-- | -- | -- | --
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x140cabcc6 | /mxe/usr/lib/gcc/x86_64-w64-mingw32.static/13.2.0/include/c++/bits/stl_iterator.h | 1077
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x140c8a8b3 | /mxe/usr/lib/gcc/x86_64-w64-mingw32.static/13.2.0/include/c++/bits/vector.tcc | 114
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x140f50e1b | rts/lib/sol2/sol.hpp | 16191
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x140fe182c | rts/lib/sol2/sol.hpp | 8398
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x1408f9c0d | rts/lib/lua/src/ldo.cpp | 321
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x14090b893 | rts/lib/lua/src/lvm.cpp | 620
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x1408fa015 | rts/lib/lua/src/ldo.cpp | 378
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x1408fa3b9 | rts/lib/lua/src/ldo.cpp | 119
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x1408f2c57 | rts/lib/lua/src/lapi.cpp | 833
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x1408f573e | rts/lib/lua/src/lbaselib.cpp | 390
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x1408f9c0d | rts/lib/lua/src/ldo.cpp | 321
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x14090b893 | rts/lib/lua/src/lvm.cpp | 620
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x1408fa015 | rts/lib/lua/src/ldo.cpp | 378
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x1408fa3b9 | rts/lib/lua/src/ldo.cpp | 119
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x1408f2c57 | rts/lib/lua/src/lapi.cpp | 833
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x14021dfaa | rts/Lua/LuaHandle.cpp | 363
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x140222c4a | rts/Lua/LuaHandle.cpp | 458
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x14022ccdb | rts/Lua/LuaHandle.h | 394
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x140709b19 | rts/System/EventHandler.cpp | 828
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x140135c27 | rts/Game/UI/MouseHandler.cpp | 513
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x14071f445 | rts/System/Input/MouseInput.cpp | 86
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x14071d870 | rts/System/Input/InputHandler.cpp | 14
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x14075b795 | /mxe/usr/lib/gcc/x86_64-w64-mingw32.static/13.2.0/include/c++/bits/basic_string.h | 223
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x140739c0b | rts/System/Main.cpp | 50
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x1400012ee | /mxe/tmp-gcc-x86_64-w64-mingw32.static/gcc-13.2.0.build_/mingw-w64-v11.0.0/mingw-w64-crt/crt/crtexe.c | 273
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x1400013e6 | /mxe/tmp-gcc-x86_64-w64-mingw32.static/gcc-13.2.0.build_/mingw-w64-v11.0.0/mingw-w64-crt/crt/crtexe.c | 164
C:\Windows\SYSTEM32\ntdll.dll | 0x000000000004cc91 | ?? | 0

</body>
</html>module	address	file	line
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x140cabcc6	/mxe/usr/lib/gcc/x86_64-w64-mingw32.static/13.2.0/include/c++/bits/stl_iterator.h 	1077
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x140c8a8b3	/mxe/usr/lib/gcc/x86_64-w64-mingw32.static/13.2.0/include/c++/bits/vector.tcc 	114
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x140f50e1b	rts/lib/sol2/sol.hpp 	[16191](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/lib/sol2/sol.hpp#L16191)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x140fe182c	rts/lib/sol2/sol.hpp 	[8398](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/lib/sol2/sol.hpp#L8398)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x1408f9c0d	rts/lib/lua/src/ldo.cpp 	[321](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/lib/lua/src/ldo.cpp#L321)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x14090b893	rts/lib/lua/src/lvm.cpp 	[620](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/lib/lua/src/lvm.cpp#L620)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x1408fa015	rts/lib/lua/src/ldo.cpp 	[378](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/lib/lua/src/ldo.cpp#L378)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x1408fa3b9	rts/lib/lua/src/ldo.cpp 	[119](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/lib/lua/src/ldo.cpp#L119)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x1408f2c57	rts/lib/lua/src/lapi.cpp 	[833](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/lib/lua/src/lapi.cpp#L833)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x1408f573e	rts/lib/lua/src/lbaselib.cpp 	[390](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/lib/lua/src/lbaselib.cpp#L390)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x1408f9c0d	rts/lib/lua/src/ldo.cpp 	[321](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/lib/lua/src/ldo.cpp#L321)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x14090b893	rts/lib/lua/src/lvm.cpp 	[620](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/lib/lua/src/lvm.cpp#L620)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x1408fa015	rts/lib/lua/src/ldo.cpp 	[378](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/lib/lua/src/ldo.cpp#L378)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x1408fa3b9	rts/lib/lua/src/ldo.cpp 	[119](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/lib/lua/src/ldo.cpp#L119)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x1408f2c57	rts/lib/lua/src/lapi.cpp 	[833](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/lib/lua/src/lapi.cpp#L833)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x14021dfaa	rts/Lua/LuaHandle.cpp 	[363](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/Lua/LuaHandle.cpp#L363)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x140222c4a	rts/Lua/LuaHandle.cpp 	[458](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/Lua/LuaHandle.cpp#L458)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x14022ccdb	rts/Lua/LuaHandle.h 	[394](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/Lua/LuaHandle.h#L394)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x140709b19	rts/System/EventHandler.cpp 	[828](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/System/EventHandler.cpp#L828)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x140135c27	rts/Game/UI/MouseHandler.cpp 	[513](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/Game/UI/MouseHandler.cpp#L513)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x14071f445	rts/System/Input/MouseInput.cpp 	[86](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/System/Input/MouseInput.cpp#L86)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x14071d870	rts/System/Input/InputHandler.cpp 	[14](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/System/Input/InputHandler.cpp#L14)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x14075b795	/mxe/usr/lib/gcc/x86_64-w64-mingw32.static/13.2.0/include/c++/bits/basic_string.h 	223
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x140739c0b	rts/System/Main.cpp 	[50](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/System/Main.cpp#L50)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x1400012ee	/mxe/tmp-gcc-x86_64-w64-mingw32.static/gcc-13.2.0.build_/mingw-w64-v11.0.0/mingw-w64-crt/crt/crtexe.c 	273
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x1400013e6	/mxe/tmp-gcc-x86_64-w64-mingw32.static/gcc-13.2.0.build_/mingw-w64-v11.0.0/mingw-w64-crt/crt/crtexe.c 	164
C:\Windows\SYSTEM32\ntdll.dll	0x000000000004cc91	?? 	0

I only changed the following code in the example:
```html
		<label>
			<h4>Range Input</h4>
			<input type="range" min="0" max="100" step="1"/>
			<h4>Range Input</h4>

			<input type="range" min="0" max="100" step="1"/>
			<input type="range" min="0" max="100" step="1"/>
		</label>

```

**Comment by Beherith** [comment]
Clicking here is what triggered the crash:

![image](https://github.com/user-attachments/assets/504566f0-7b1c-455f-8bec-ddfcb8617b7d)

**Comment by Beherith** [comment]
Hmm, another crash on exit:

<html><body>

C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x140cabcc6 | /mxe/usr/lib/gcc/x86_64-w64-mingw32.static/13.2.0/include/c++/bits/stl_iterator.h | 1077
-- | -- | -- | --
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x140c8a8b3 | /mxe/usr/lib/gcc/x86_64-w64-mingw32.static/13.2.0/include/c++/bits/vector.tcc | 114
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x140f50e1b | rts/lib/sol2/sol.hpp | 16191
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x140fe182c | rts/lib/sol2/sol.hpp | 8398
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x1408f9c0d | rts/lib/lua/src/ldo.cpp | 321
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x14090b893 | rts/lib/lua/src/lvm.cpp | 620
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x1408fa015 | rts/lib/lua/src/ldo.cpp | 378
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x1408fa3b9 | rts/lib/lua/src/ldo.cpp | 119
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x1408f2c57 | rts/lib/lua/src/lapi.cpp | 833
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x1408f573e | rts/lib/lua/src/lbaselib.cpp | 390
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x1408f9c0d | rts/lib/lua/src/ldo.cpp | 321
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x14090b893 | rts/lib/lua/src/lvm.cpp | 620
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x1408fa015 | rts/lib/lua/src/ldo.cpp | 378
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x1408fa3b9 | rts/lib/lua/src/ldo.cpp | 119
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x1408f2c57 | rts/lib/lua/src/lapi.cpp | 833
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x14021dfaa | rts/Lua/LuaHandle.cpp | 363
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x140222c4a | rts/Lua/LuaHandle.cpp | 458
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x140222eae | rts/Lua/LuaHandle.cpp | 534
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x140222fc8 | rts/Lua/LuaHandle.cpp | 178
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x1402e5369 | rts/Lua/LuaUI.cpp | 63
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x14003b93e | rts/Game/Game.cpp | 958
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x140756def | rts/System/SpringApp.cpp | 969
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x14075b9ae | rts/System/Platform/Threading.h | 188
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x140739c0b | rts/System/Main.cpp | 50
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x1400012ee | /mxe/tmp-gcc-x86_64-w64-mingw32.static/gcc-13.2.0.build_/mingw-w64-v11.0.0/mingw-w64-crt/crt/crtexe.c | 273
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe | 0x1400013e6 | /mxe/tmp-gcc-x86_64-w64-mingw32.static/gcc-13.2.0.build_/mingw-w64-v11.0.0/mingw-w64-crt/crt/crtexe.c | 164


</body>
</html>C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x140cabcc6	/mxe/usr/lib/gcc/x86_64-w64-mingw32.static/13.2.0/include/c++/bits/stl_iterator.h 	1077
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x140c8a8b3	/mxe/usr/lib/gcc/x86_64-w64-mingw32.static/13.2.0/include/c++/bits/vector.tcc 	114
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x140f50e1b	rts/lib/sol2/sol.hpp 	[16191](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/lib/sol2/sol.hpp#L16191)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x140fe182c	rts/lib/sol2/sol.hpp 	[8398](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/lib/sol2/sol.hpp#L8398)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x1408f9c0d	rts/lib/lua/src/ldo.cpp 	[321](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/lib/lua/src/ldo.cpp#L321)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x14090b893	rts/lib/lua/src/lvm.cpp 	[620](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/lib/lua/src/lvm.cpp#L620)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x1408fa015	rts/lib/lua/src/ldo.cpp 	[378](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/lib/lua/src/ldo.cpp#L378)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x1408fa3b9	rts/lib/lua/src/ldo.cpp 	[119](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/lib/lua/src/ldo.cpp#L119)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x1408f2c57	rts/lib/lua/src/lapi.cpp 	[833](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/lib/lua/src/lapi.cpp#L833)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x1408f573e	rts/lib/lua/src/lbaselib.cpp 	[390](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/lib/lua/src/lbaselib.cpp#L390)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x1408f9c0d	rts/lib/lua/src/ldo.cpp 	[321](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/lib/lua/src/ldo.cpp#L321)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x14090b893	rts/lib/lua/src/lvm.cpp 	[620](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/lib/lua/src/lvm.cpp#L620)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x1408fa015	rts/lib/lua/src/ldo.cpp 	[378](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/lib/lua/src/ldo.cpp#L378)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x1408fa3b9	rts/lib/lua/src/ldo.cpp 	[119](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/lib/lua/src/ldo.cpp#L119)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x1408f2c57	rts/lib/lua/src/lapi.cpp 	[833](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/lib/lua/src/lapi.cpp#L833)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x14021dfaa	rts/Lua/LuaHandle.cpp 	[363](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/Lua/LuaHandle.cpp#L363)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x140222c4a	rts/Lua/LuaHandle.cpp 	[458](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/Lua/LuaHandle.cpp#L458)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x140222eae	rts/Lua/LuaHandle.cpp 	[534](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/Lua/LuaHandle.cpp#L534)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x140222fc8	rts/Lua/LuaHandle.cpp 	[178](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/Lua/LuaHandle.cpp#L178)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x1402e5369	rts/Lua/LuaUI.cpp 	[63](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/Lua/LuaUI.cpp#L63)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x14003b93e	rts/Game/Game.cpp 	[958](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/Game/Game.cpp#L958)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x140756def	rts/System/SpringApp.cpp 	[969](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/System/SpringApp.cpp#L969)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x14075b9ae	rts/System/Platform/Threading.h 	[188](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/System/Platform/Threading.h#L188)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x140739c0b	rts/System/Main.cpp 	[50](http://github.com/beyond-all-reason/spring/tree/BAR105/rts/System/Main.cpp#L50)
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x1400012ee	/mxe/tmp-gcc-x86_64-w64-mingw32.static/gcc-13.2.0.build_/mingw-w64-v11.0.0/mingw-w64-crt/crt/crtexe.c 	273
C:\Users\user\AppData\Local\Programs\Beyond-All-Reason\data\engine\105.1.1-2708-g3d35645 bar\spring.exe	0x1400013e6	/mxe/tmp-gcc-x86_64-w64-mingw32.static/gcc-13.2.0.build_/mingw-w64-v11.0.0/mingw-w64-crt/crt/crtexe.c 	164

**Comment by saurtron** [comment]
could be related to this PR: https://github.com/beyond-all-reason/spring/pull/1761

can you try with that to see if it helps?

**Comment by Beherith** [comment]
Is there a release build of that or?

**Comment by saurtron** [comment]
> Is there a release build of that pr?

No. But if it's not practical for you don't worry, I'll see if I can reproduce the error and try the branch.

**Comment by ChrisFloofyKitsune** [comment]
fixed in #1761
